### PR TITLE
Fifth elixir kata

### PR DESCRIPTION
**Pull request by thelastinuit**

### CODE REVIEW

File: thelastinuit/21-aug-triple_trouble.exs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# https://www.codewars.com/kata/5704aea738428f4d30000914
+
+defmodule Codewars.WeirdString do
+  def triple_trouble(one, two, three) do
+    # For fancy chars, use graphemes. it's slow, though.
+    chars_one = one |> String.codepoints
```
Comment (thelastinuit) [inline thelastinuit/21-aug-triple_trouble.exs]: I thought this was sort of terrible but seeing other solutions doesn't look so terrible.

Comment (mayra-cabrera) [inline thelastinuit/21-aug-triple_trouble.exs]: huh! I have never seen that symbol before, what  `|>` it is supposed to be doing?

Comment (thelastinuit) [inline thelastinuit/21-aug-triple_trouble.exs]: piping biatch!!

Comment (mayra-cabrera) [inline thelastinuit/21-aug-triple_trouble.exs]: can you be a bit more specific? 🙄

Comment (thelastinuit) [inline thelastinuit/21-aug-triple_trouble.exs]: piping biatch is translated from cejudaismo to english as
The pipe operator which looks like |> takes the result of one expression, and passes it on.

check it out here: https://elixirschool.com/en/lessons/basics/pipe-operator/

**Pull request merged**